packages/components/src/color-picker/test/__snapshots__/index.js.snap

```diff
@@ -1,968 +1,576 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ColorPicker should commit changes to all views on blur 1`] = `
-.emotion-4 {
-  font-family: -apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen-Sans,Ubuntu,Cantarell,'Helvetica Neue',sans-serif;
-  font-size: 13px;
-}
-
-.emotion-2 {
-  margin-bottom: 8px;
-}
-
-.components-panel__row .emotion-2 {
-  margin-bottom: inherit;
-}
-
-.emotion-0 {
-  display: inline-block;
-  margin-bottom: 8px;
-}
+"Snapshot Diff:
```

Comment:
Is it possible to change those hints in the output? I'm nitpicking in general, but it would better explain the intent if there was something like:
```
"Snapshot Diff:
- Base component
- Current component
```